Src/LinearSolvers/CMakeLists.txt

```diff
@@ -98,3 +98,14 @@ if (AMReX_HYPRE)
       MLMG/AMReX_MLNodeLaplacian_hypre.cpp
       )
 endif ()
+
+if (AMReX_SPACEDIM EQUAL 3)
+
+   target_include_directories(amrex PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_LIST_DIR}/OpenBC>)
+
+   target_sources(amrex
+      PRIVATE
+      OpenBC/AMReX_OpenBC.H
```

Comment:
We probably don't really need it because that file is only for AMReX_OpenBC.cpp, not for public.  But it's probably less likely to miss a public header if we simply include everything.  @ax3l What's your opinion?

Reply:
Yes, usually public API is what you like to add. But if you add all headers, nothing will go wrong either.

Reply:
Thanks.  I added the header.